Plugins/SwiftLintCommandPlugin/SwiftLintCommandPlugin.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import PackagePlugin
+
+@main
+internal struct SwiftLintCommandPlugin: CommandPlugin {
+    internal func performCommand(
```

Comment:
May we take advantage of implicit internal?
```suggestion
struct SwiftLintCommandPlugin: CommandPlugin {
    func performCommand(
```

Reply:
Oh yeah, please. 👍